src/include/duckdb/common/types/value.hpp

```diff
@@ -69,6 +69,10 @@ class Value {
 		return is_null;
 	}
 
+	inline void SetNull() {
```

Comment:
Is this necessary? Values are supposed to be mostly immutable nowadays.

Reply:
No I don't think it is, I created it because at the time I didn't realize that Value::Value(LogicalType) creates a NULL Value of a given type